kinesumer.go

```diff
@@ -488,30 +494,37 @@ func (k *Kinesumer) consumePipe(stream string, shard *Shard) {
 				}
 			}
 		case <-checkPointTicker.C:
-			k.commitCheckPoint(stream, shard.ID, lastSequence)
+			k.Commit(stream, shard.ID, lastSequence)
 		}
```

Comment:
By default, Kinesumer is always doing auto commit. If you want to provide commit option (auto or manual), you should also provide the commit option configuration. (ex. auto commit flag) And need to update commit logics to handle both cases.